cmd/buildah/images.go

```diff
@@ -24,22 +46,35 @@ var (
 			Usage: "do not print column headings",
 		},
 		cli.BoolFlag{
-			Name:  "notruncate",
+			Name:  "no-trunc, notruncate",
```

Comment:
What's the rationale for adding this alias for this option to this command, but not to any of the other commands that offer a `--notruncate` option?